oak_functions/examples/key_value_lookup/module/src/tests.rs

```diff
@@ -33,16 +33,15 @@ async fn test_server() {
         }),
     );
 
-    let server_port = oak_functions_test_utils::free_port();
-    let server_background = oak_functions_test_utils::create_and_start_oak_functions_server(
-        server_port,
-        &wasm_path,
-        lookup_data_file.path().to_str().unwrap(),
-    )
-    .unwrap();
+    let (_server_background, server_port) =
+        xtask::launcher::run_oak_functions_example_in_background(
+            &wasm_path,
+            lookup_data_file.path().to_str().unwrap(),
+        )
+        .await;
 
     // Wait for the server to start up.
-    std::thread::sleep(Duration::from_secs(2));
+    std::thread::sleep(Duration::from_secs(20));
```

Comment:
We should probably move that magic value of seconds into a constant and inline it into `xtask::launcher::run_oak_functions_example_in_background`. At the moment we manually paste this line below every invocation of that function. 

Will do so in a seperate refactoring PR